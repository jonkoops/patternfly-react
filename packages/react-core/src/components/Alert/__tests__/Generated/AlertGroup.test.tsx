/**
 * @vitest-environment jsdom
 */
import { render } from '@testing-library/react';
import React from 'react';
import { expect, it } from 'vitest';

import { AlertGroup } from '../../AlertGroup';

it('AlertGroup should match snapshot (auto-generated)', () => {
  const { asFragment } = render(
    <AlertGroup className={'string'} children={<div>ReactNode</div>} isToast={true} appendTo={undefined} />
  );
  expect(asFragment()).toMatchSnapshot();
});
