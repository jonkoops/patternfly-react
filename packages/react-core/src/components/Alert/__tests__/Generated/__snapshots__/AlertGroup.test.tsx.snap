// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AlertGroup should match snapshot (auto-generated) 1`] = `<DocumentFragment />`;
